servicetalk-examples/docs/modules/ROOT/pages/http/index.adoc

```diff
@@ -73,10 +73,22 @@ blocking iterable stream of buffers.
 Extends the async "Hello World" example to demonstrate content encoding compression filters. No separate example is
 needed for the other API variants as the usage of content encoding filters is the same for all API styles.
 
-* link:{source-root}/servicetalk-examples/http/compression/src/main/java/io/servicetalk/examples/http/compression/CompressionFilterExampleServer.java[CompressionFilterExampleServer] - a server that demonstrates 
+* link:{source-root}/servicetalk-examples/http/compression/src/main/java/io/servicetalk/examples/http/compression/CompressionFilterExampleServer.java[CompressionFilterExampleServer] - a server that demonstrates
 the asynchronous API and responds with a simple `Hello World!` response body, optionally customized for a specific name for `POST` requests, as a `text/plain`.
 * link:{source-root}/servicetalk-examples/http/compression/src/main/java/io/servicetalk/examples/http/compression/CompressionFilterExampleClient.java[CompressionFilterExampleClient.java] - a client that
-sends a `POST` request containing a name to the link:{source-root}/servicetalk-examples/http/compression/src/main/java/io/servicetalk/examples/http/compression/CompressionFilterExampleServer.java[server] and 
+sends a `POST` request containing a name to the link:{source-root}/servicetalk-examples/http/compression/src/main/java/io/servicetalk/examples/http/compression/CompressionFilterExampleServer.java[server] and
+receives a response greeting the posted name as a single content.
+
+[#Debugging]
+== Debugging
+
+Extends the async "Hello World" example to demonstrate some debugging features available
+ for debugging ServiceTalk applications.
+
+* link:{source-root}/servicetalk-examples/http/debugging/src/main/java/io/servicetalk/examples/http/debugging/DebuggingExampleServer.java[DebuggingExampleServer] - a server that demonstrates
+the asynchronous API and responds with a simple `Hello World!` response body, optionally customized for a specific name for `POST` requests, as a `text/plain`.
```

Comment:
Consider clarifying the real purpose of this example - it demonstrates debuggability features, it doesn't matter what API it uses or what type or request sends.